src/Sulu/Bundle/CategoryBundle/Command/RecoverCommand.php

```diff
@@ -17,15 +17,15 @@
 use Symfony\Component\Console\Input\InputInterface;
 use Symfony\Component\Console\Input\InputOption;
 use Symfony\Component\Console\Output\OutputInterface;
+use Symfony\Component\Console\Attribute\AsCommand;
 
 /**
  * Command for recovering categories.
  * This command is fixing wrong left/right and depths (see -d) assignments of the categories tree.
  */
+#[AsCommand(name: 'sulu:categories:recover')]
```

Comment:
Is this now the only way to define this or can it be defined this?

If I read the docs about it, it maybe make also sense to move descriptions to it?